server/internal/graphql/model/models_gen.go

```diff
@@ -69,8 +69,10 @@ type CatalogPattern struct {
 }
 
 type CatalogSelector struct {
-	Search string `json:"search"`
-	Order  string `json:"order"`
+	Page     string `json:"page"`
+	PageSize string `json:"pageSize"`
```

Comment:
Let's be consistent with `json` tags as well. `page_size`. You can update GraphQL schema in a way that this tag is generated appropriately.

Reply:
@MUzairS15 done

Reply:
@MUzairS15 What do you mean by relay scripts ? Can you point them out to me please.

Reply:
https://github.com/meshery/meshery/blob/cc6073c2a8e1aeb5d94e7940aa6422f4409978d5/ui/components/graphql
Run really script as follows: `cd ui; npm run relay;`

Reply:
ran relay script